types/node/scripts/generate-docs/html-doc-processing.ts

```diff
@@ -120,6 +120,9 @@ export function fixupHtmlDocs(input: string, context: DocContext): string {
         input = input.replace(/<pre><code class="language-cjs">.*?<\/code><\/pre>/igms, '');
     }
 
+    // workaround markdown generation bug
+    input = input.replaceAll(/([^ ])\n<em>/g, '$1 \n<em>');
```

Comment:
What's this script used for and what is this line supposed to do?

Reply:
@SimonSchick Simon is the original owner of the implementation (sorry for direct ping Simon)

Reply:
I think this was just suppose to strip whitespaces, I can't remember the exact logic here..

Reply:
This line detects where there's a `<newline>` `<em>` that isnt preceded by a space, and adds a space. The markdown generation seems to drop the spacing when it's a newline, so this is a workaround.